frontend/src/components/Header.tsx

```diff
@@ -152,6 +184,11 @@ const NavLinks = ({
     }
   }, [mobileExpanded, onToggleMobileNav]);
 
+  const closeDropdownAndMobileNav = useCallback(() => {
```

Comment:
I'm not sure this is necessary - the mobile nav will only be relevant at mobile viewport widths, and the dropdowns will only be relevant at > mobile widths. Am I missing a case where these could be relevant at the same time? I was thinking this may be in place to close the dropdowns if the window is resized but that doesn't seem to be happening.

Reply:
@doug-s-nava added this because both instances of `NavLink` in this file had `closeMobileNav` passed as the `onClick` prop and I needed to also pass `setActiveNavDropdown(null)` as the onClick action for both to close any open `NavDropdown`s when a non-navdropdown nav item is clicked. This function makes is so that I can cleanly pass `closeDropdownAndMobileNav` as both NavLink's on click actions, as opposed to: 
```
onClick = {() => {
            closeMobileNav;
            setActiveNavDropdownIndex(null);
          }}
```

But I could remove this function and opt for the above instead if that is your preference!

Reply:
Gotcha, thanks for the explanation. I'm good with this! It may not be necessary after the rebase to pull in my PR to auto close the sub nav, but we can revisit after you've pulled that in